tools/protocol-schema-check/res/protocol_schema.toml

```diff
@@ -168,6 +168,7 @@ MaybeEncodedShardChunk = 1535469720
 MerklePathItem = 2615629611
 MessageDiscriminant = 3240833245
 MethodResolveError = 1206790835
+MissingTrieValue = 856208857
```

Comment:
@saketh-are Continuing the discussion https://github.com/near/nearcore/pull/12961#discussion_r2064454065 here. After several iterations we came down to just one change in the protocol_schema.toml

Do you believe it is still a problem? :-(